### PR TITLE
Feature/aws deploy efs

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR completes the work started by @pcm32 in making an automated deployment of the Hinxton single-cell Galaxy setup on AWS. 

Scripts are provided to build the cluster and connected storage via EFS, and for teardown of all the associated components (security groups etc).

Expect some tweaking to be required before any PR to upstream repo.

### CODE REVIEW

File: deployment-guides/AWS-EKS-EFS/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+### Environment
+
+Edit `<checkout_dir>/deployment-guides/AWS-EKS-EFS/destroy-all.sh`. In particular you'll need to modify DEVELOPER and AWS_REGION:
+
+```
+export DEVELOPER=<your amazon account ID>
+export AWS_REGION=<an Amazon region, e.g. us-west-2>
+```
```
Comment (pcm32) [inline deployment-guides/AWS-EKS-EFS/README.md:23]: Why are the above settings (authentication) on the destroy-all.sh file? shouldn't they be in a general config file outside of the version controlled ones - you could have a .sample for instance.

Comment (pcm32) [reply deployment-guides/AWS-EKS-EFS/README.md:23]: I guess that by developer, you mean an AWS key, right? Could you also comment on the authorisation level (policies) that that user needs to have access to execute all of these? It must be the PowerUser role I guess.

Comment (pcm32) [reply deployment-guides/AWS-EKS-EFS/README.md:23]: Ahh, I see that they are in an env file, is just the docs here that need polishing. You probably want to mention here as well that the aws-cli needs to be authenticated with an aws key (with the adequate permissions).

**Review by pcm32 — COMMENTED** [review]
Very nice work! Some comments only on the documentation part. I wonder if we want to remove the FSx variant and leave it in a different branch since it is not finished? Did you have the chance of adding the throughput setting for EFS?

File: deployment-guides/AWS-EKS-EFS/README.md (hunk 1 above)
Comment (pinin4fjords) [reply deployment-guides/AWS-EKS-EFS/README.md:23]: So, 'developer' is the key you need to pass to 'aws efs create-file-system', hence the variable name- have corrected the README file

**Comment by pinin4fjords** [comment]
Thanks for the comments @pcm32 ! For the throughput, the guidelines say you should run under GeneralPurpose for a while, then use the usage stats to decide on what other settings are required. So I've left it for now, with a view to tweaking at such time as we actually deploy this in anger.

**Comment by pinin4fjords** [comment]
Agreed on fsx- should I rebase to develop @pcm32 ?

**Comment by pcm32** [comment]
Yeah, sounds good, whatever you prefer...

**Comment by pinin4fjords** [comment]
Sorry, bit messy, but couldn't get rebase etc to play ball, so just removed the files from this branch.

**Pull request merged**